Source/Krypton Components/Krypton.Toolkit/Utilities/PaletteImageScaler.cs

```diff
@@ -38,14 +38,12 @@ public static class PaletteImageScaler
         /// <param name="pal">KryptonPalette</param>
         public static void ScalePalette(float factorDpiX, float factorDpiY, KryptonPalette pal)
         {
-            if (pal == null
-//            || pal.HasAlreadyBeenScaled
-                )
+            if (pal == null /*|| pal.HasAlreadyBeenScaled*/)
```

Comment:
keep conditions on separate lines, it is then easier to comment out, put back in to test

Reply:
Done